contributing/code/standards.rst

```diff
@@ -234,14 +248,17 @@ Documentation
 
 * Omit the ``@return`` tag if the method does not return anything;
 
-* The ``@package`` and ``@subpackage`` annotations are not used.
+* The ``@package`` and ``@subpackage`` annotations are not used;
+
+* Inline the ``@inheritdoc`` tag.
 
 License
--------
+~~~~~~~
 
 * Symfony is released under the MIT license, and the license block has to be
   present at the top of every PHP file, before the namespace.
 
+.. _`PHP CS Fixer tool`: https://github.com/FriendsOfPHP/PHP-CS-Fixer
```

Comment:
everywhere there is website, here you put repo
please use `cs.sensiolabs.org` for consistency